.github/ISSUE_TEMPLATE/feature_suggestion.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Feature suggestion
+about: Suggest an idea for this project
+
+---
+<!--- Please fill out this template to the best of your ability. You can always edit this issue once you have created it. -->
+## Feature suggestion
+A suggestion that is not clearly explained or is duplicate to a previous one will be closed
+**Describe the feature suggestion**
+A clear and concise description of what the suggestion is. 
+<!--- Put your text below this line -->
+
+
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+<!--- Put your text below this line -->
+
+
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+<!--- Put your text below this line -->
+
+
+
+**Additional context or screenshots (if appropriate):**
+Add any other context or screenshots about the feature request here.
+<!--- Put your text below this line -->
+
+
+
+
+<!--- End of this issue -->
+*Note: Team Kodi will consider your request however we will not make any promises if it will be included at some point in the future.*
```

Comment:
ditto

Reply:
Should feature requests even be considered for submission via github? I was aware we were going to drop trac in favor or github issues but feature requests aren't likely to help development and will only generate entropy. We already have a specific forum section dedicated to this. Ignore if I somehow missed that discussion.

Reply:
We at least agreed (I think?) to use it for stuff that team members feel should be done at some point, or that they themselves want to do in the future etc. - so as TODO, roadmap, or whatever you want to call it.

+1 for not having all sorts of general user feature requests in varying stages of concreteness here though. Maybe the forum is a better place for those.

Reply:
Will change it to roadmap with the note it's for team only